src/isilon_hadoop_tools/onefs.py

```diff
@@ -288,6 +288,7 @@ class APIError(_BaseAPIError):
     user_not_found_error_format = "Failed to find user for 'USER:{0}': No such user"
     user_unresolvable_error_format = "Could not resolve user {0}"
     zone_not_found_error_format = 'Access Zone "{0}" not found.'
+    directory_path_already_exists_error_format = 'Unable to create directory as requested -- container already exists'
```

Comment:
https://travis-ci.com/github/Isilon/isilon_hadoop_tools/jobs/319561783#L322
```suggestion
    directory_path_already_exists_error_format = \
        'Unable to create directory as requested -- container already exists'
```